### ISSUE TITLE
How to remove document margin?

### ISSUE THREAD

**Issue by kazzkiq** [RelaxedJS/ReLaXed#68]
I'm trying to remove document margin, so I can paint/handle it myself accordingly to the page I'm on. But it seems the only way to do it is adding negative margin to `@page` as `margin: 0mm;` doesn't works at all.

How can I fully remove the document margin?

![captura de tela 2018-05-27 as 02 06 06](https://user-images.githubusercontent.com/1953194/40582628-8c1b400c-6152-11e8-8bed-c5e8553724d0.png)

**Comment by kazzkiq** [comment]
I just figured that `body { margin: 0 }` does the trick.

However, the generated PDF seems to keep a white stripe at the right side. Any ideas how to get rid of that?

**Comment by Zulko** [comment]
``html { margin: 0; padding: 0}`` ?

**Comment by kazzkiq** [comment]
It doesn't seem to fix the slim white strip on the right side (see picture in my last comment). Perhaps a macOS bug?

**Comment by DanielRuf** [comment]
Do you have a example html document for us which we can use to reproduce / test this?

**Comment by kazzkiq** [comment]
Yes. You can check the files I've used below:

<details>
  <summary>book.pug</summary>

  ```pug
  #cover
    h1.title Hello

  style
    include:scss theme.scss
  ```

</details>

-----

<details>
  <summary>theme.scss</summary>

  ```sass
  @import 'variables';

  @page {
    margin: 0;
    size: $page_width $page_height;
    -relaxed-page-width: $page_width;
    -relaxed-page-height: $page_height;
  }

  html, body {
    margin: 0;
    width: $page_width;
    height: $page_height;
    font-family: $font-family;
  }

  #cover {
    width: $page_width;
    height: $page_height;
    background: $color-blue;
    color: #fff;
    padding: 10mm;
  }
  ```

</details>

-----

<details>
  <summary>variables.scss</summary>

  ```sass

$color-blue: #233142;
$color-gold: #FCD08A;
$color-gold-dark: #6F483F;

$font-family: serif;


// A4
$page_width: 210mm;
$page_height: 297mm;
  ```

</details>

**Comment by Zulko** [comment]
another approach is: when you try [this example](https://github.com/RelaxedJS/ReLaXed-examples/tree/master/examples/business-card), do you see a margin on the left ?

**Comment by kazzkiq** [comment]
>@Zulko: when you try this example, do you see a margin on the left ?

Nope. But as soon as I change the size to A4 (210mm x 297mm) the white stripe appears:

```sass
@page {
  size: 210mm 297mm;
  -relaxed-page-width: 210mm; 
  -relaxed-page-height: 297mm;
  margin: 0;
  padding: 0;
}
```

PDF (check right-side for the white stripe):

![captura de tela 2018-06-02 as 22 59 02](https://user-images.githubusercontent.com/1953194/40882445-9a5f994a-66b8-11e8-92ea-15e4a72cdf3b.png)

**Comment by Zulko** [comment]
Interesting, might be a float precision. For A4 pages it should be sufficient to not give any size parameters (remove ``size`` and ``-relaxed-`` parameters) since A4 is the default. Maybe it will work better.

**Comment by gueorgui** [comment]
Has there been any updates on the right side white stripe issue? Just betting on A4 working well because it's the default doesn't solve my use case unfortunately.

**Comment by fro** [comment]
No matter the parameters, nothing can prevent the white strip to appear up, right, down. Here's what I have tried:

```
@page {
  // default
}

@page {
  margin: 0px 0px 0px 0px;
}

@page {
  -relaxed-page-size: A4;
  size: A4;
}

@page {
  -relaxed-page-width: 210mm;
  -relaxed-page-height: 297mm;
  size: 210mm 297mm;
}

@page {
  -relaxed-page-width: 210mm;
  -relaxed-page-height: 297mm;
  -relaxed-page-size: A4;
  size: A4;
  margin: 0px 0px 0px 0px;
}

@page {
  -relaxed-page-width: 210mm;
  -relaxed-page-height: 297mm;
  size: 210mm 297mm;
  margin: 0px 0px 0px 0px;
}
```

The best I can do is **21,01 × 29,71 cm** instead of **210 mm × 297 mm**.

My two cents, @Zulko is right about the float precision.
